cityofaustin/atd-moped
# Fixes advanced search filter lag and adds query caching

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/14257
-  https://github.com/cityofaustin/atd-data-tech/issues/10799

## Testing
**URL to test:** [Netlify](https://deploy-preview-1167--atd-moped-main.netlify.app/)

1. Load the project list view.
2. Navigate to any project except the first project in the project list. 
3. Edit the project description
4. Navigate back to the project list—watch your project jump to the top of the project list
5. Repeat those steps with another project, but this time edit the project description, save, wait 10 seconds, then edit the description again.
6. Navigate back to the project list—your edited project should already be at the top of the project list. This demonstrates the race condition wrt to a project's `updated_at` date. 
7. Increase the rows per page to 1,000, then open the advanced search. It should open without any lag!
8. Apply some advanced search criteria, then navigate to a project and use the `< ALL PROJECTS` link to navigate back to the project list with your advanced filters applied.
9. Test that column sorting and pagination work normally.
10. Add, edit, and map project components. This ensures that the new component object cloning has not broken anything.

I think that's it! 😅 
---
#### Ship list
